assets/helpers/tracking/conversions.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+import { getAbsoluteURL } from '../url';
+import { pageView } from './ophanComponentEventTracking';
+import { successfulConversion } from './googleTagManager';
+import type { Participations } from '../abTests/abtest';
+
+export default function trackConversion(
+  participations: Participations,
+  currentRoute: string,
+) {
+  // Fire GTM conversion events
+  successfulConversion(participations);
+  // Send an Ophan pageview. Because this function is used to track page views
+  // from client side routed thank you pages, the referrer will always be the current location
+  pageView(getAbsoluteURL(currentRoute), document.location.href);
```

Comment:
Minor point, but doesn't this rely on this function knowing where it's going to be called from? Would it make sense to pass through the referrer as well and pass it `document.location.href` explicitly?

Reply:
I did that originally but it seemed redundant as it was always the same